config/order.php

```diff
@@ -6,4 +6,5 @@
             'rejected' => 3,
             'cancelled' => 4,
         ],
+        "message_pending" => "message_pending",
```

Comment:
```suggestion
        "message_pending" => 0,
```
đặt config như này chứ

Reply:
Cái message_pending này e có lưu vào i18n, mà để vào trong code em thấy không hợp lý lên để vào config ạ.

Reply:
Lúc tạo data để gửi thông báo ạ. em có lưu key của i18n ở trong code. mà code thì không lên để để số hay chuỗi lên em viết ở config ạ.